Idle and walking animations added to mouse

PaulTashima opened the pull request:
Adds animator to Mouse and new sprite sheet. Replaces sprite manipulation code in MouseController with blend trees in the animator.

buzzbuzz55 committed:
Idle and walking animations added to mouse

Adds animator to Mouse and new sprite sheet. Replaces sprite manipulation code in MouseController with blend trees in the animator.

rhonig22 commented on Assets/Scripts/Player/PlayerController.cs:
```diff
@@ -23,6 +23,7 @@ public class PlayerController : MonoBehaviour
     [SerializeField] private Sprite _mouseFront;
```
Remove these, as they are no longer needed.

rhonig22 commented on Assets/Scripts/Player/PlayerController.cs:
```diff
@@ -100,37 +111,16 @@ private void Move(Vector3 targetDirection)
         Vector3 targetVelocity = targetDirection * _currentSpeed;
         if (targetDirection.magnitude > 0)
         {
+            /*  Flag for deletion. Currently changing sprite direction as a blend tree in the animator - we can do this as an animator parametre if needed
```
You can go ahead and remove this.

rhonig22 commented on Assets/Scripts/Player/PlayerController.cs:
```diff
@@ -73,6 +74,16 @@ void Update()
         _verticalInput = Input.GetAxisRaw("Vertical");
+        
+        // Animator updates; does not rewrite direction on zero to avoid changing sprite direction when input is released
+        if(_horizontalInput != 0 || _verticalInput != 0){
```
Can you break this out into a method that takes horizontal and verticalinputs as parameters?

rhonig22 commented on Assets/Sprites/Mouse_walking.png.meta:
```diff
@@ -0,0 +1,844 @@
+  spriteMeshType: 1
+  alignment: 0
+  spritePivot: {x: 0.5, y: 0.5}
+  spritePixelsToUnits: 50
```
I think using 50 px per unity is causing a few rendering issues with the sprite. I am playing around with some changes that might fix it, but it will effect the resolution of the screen.

rhonig22 reviewed (changes requested):
There are a few issues with the sprite rendering that I think should be addressed before we check this in.

rhonig22 committed:
PR fixes

- some code cleanup in the player
- change game resolution to 900x600 so it is divisible by 50
- change camera size to use 50 pixels per unit
